expo-app/components/ui/gluestack-ui-provider/index.web.tsx

```
@@ -9,87 +9,86 @@ import { ModeType } from './types';
-        }}
-      />
-      <OverlayProvider>
-        <ToastProvider>{props.children}</ToastProvider>
-      </OverlayProvider>
-    </>
-  );
+	return (
+		<>
+			<script
+				suppressHydrationWarning
+				dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)